### ISSUE TITLE
Realtime fails with multiple replications

### ISSUE THREAD

**Issue by pzhine** [marceljuenemann/rxdb-supabase#10]
Hi, I'm having the same problem described by @jmacindoe in #9 - I have 2 replications, and I'm only getting realtime updates on the second one. I checked that I am running the code with their fix (v1.0.1). When I check the network log in devtools, I do not see messages when I make updates to the table referenced by the first replication. If I comment out the second replication, everything works as expected on the first. Am I missing something, or does the bug persist?

**Comment by jmacindoe** [comment]
I think there might have been a problem with the v1.0.1 build so the latest changes weren't actually included

**Comment by pzhine** [comment]
Oh, you're right! The Typescript file in the package has your fix...

```typescript
private watchPostgresChanges() {
    this.realtimeChannel = this.options.supabaseClient
      .channel(`rxdb-supabase-${this.replicationIdentifierHash}`)
```

...but the .js file in the build dir does not

```js
watchPostgresChanges() {
    this.realtimeChannel = this.options.supabaseClient.channel("any").on("postgres_changes", { event: "*", schema: "public", table: this.table }, (payload) => {
```

Everything works as expected when I copy `supabase-replication.ts` into my project. In any case, @marceljuenemann, please make another release to fix the build when you get a chance.

**Comment by marceljuenemann** [comment]
This should be fixed in v1.0.2 once and for all!
